src/Moq/Properties/Resources.Designer.cs

```diff
@@ -572,7 +572,7 @@ internal class Resources {
         }
         
         /// <summary>
-        ///   Looks up a localized string similar to Type to mock must be an interface, a delegate, or a non-sealed, non-static class..
+        ///   Looks up a localized string similar to Mocked type {0} must be an interface, a delegate, or a non-sealed, non-static class..
```

Comment:
Please have VS regenerate this file so that it is in sync with `Resources.resx`.